<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_task</class>
 <widget class="QWidget" name="Add_task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="getDate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;添加任务&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择任务的提醒日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择任务的提醒时刻：</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="getTime">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入任务名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="getName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择任务类型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="getType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>默认：life</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>study</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>play</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>life</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>work</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择任务优先级：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="getPrior">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>默认：middle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>low</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>middle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>high</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>emergent</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
